string bin_id
---
bool success
string message